DESCRIPTION

```diff
@@ -19,7 +19,8 @@ Authors@R: c(
     person("Pradeep", "Adhokshaja",  role = "ctb"),
     person("Olivia", "Barrows", role = "ctb"),
     person("Jojo", "Miller", role = "ctb"),
-    person("Jayla", "Nakayama", role = "ctb"))
+    person("Jayla", "Nakayama", role = "ctb")),
+    person("Rana", "Gahwagy", role="ctb")
```

Comment:
* This is alas going to cause an error because of the double parentheses and the comma after Jayla's name. Could you re-do this so that comma's separate persons, and a double parentheses ends the list of persons?
* I noticed you didn't add an email. Just making sure this was a conscious choice. If not, please consider including an email that you know you'll be using after you leave Smith.